extensions/amp-image-lightbox/0.1/amp-image-lightbox.js

```diff
@@ -289,7 +294,7 @@ export class ImageViewer {
     // and then naturally upgrade to a higher quality image.
     return timerFor(this.win).promise(1).then(() => {
       this.image_.setAttribute('src', src);
-      return this.loadPromise(this.image_);
```

Comment:
yep thats a good idea 👍, this (loadPromise) probably should have been a static method instead of an instance method